ibis/backends/flink/__init__.py

```diff
@@ -145,6 +149,34 @@ def list_tables(
         # but executing the SQL string directly yields a `TableResult` object
         return self._filter_with_like(tables, like)
 
+    def list_views(
+        self,
+        like: str | None = None,
+        database: str | None = None,
+        catalog: str | None = None,
+    ) -> list[str]:
+        """Return the list of view names.
+
+        Return the list of view names in the specified database and catalog.
+        or the default one if no database/catalog is specified.
+
+        Parameters
+        ----------
+        like : str, optional
+            A pattern in Python's regex format.
+        database : str, optional
+            The database to list tables of, if not the current one.
+        catalog : str, optional
+            The catalog to list tables of, if not the current one.
```

Comment:
As my other comment alludes to, we probably want to include the views in the output of `list_tables`, but if this stays in, the docstring should be fixed to refer to `views`

Reply:
Thanks, included `views` in `list_tables()`.